packages/iris-grid/src/IrisGridTableModel.ts

```diff
@@ -80,15 +78,17 @@ class IrisGridTableModel extends IrisGridTableModelTemplate<Table, UIRow> {
   }
 
   getMemoizedFrontColumns = memoize(
-    layoutHintsFrontColumns => layoutHintsFrontColumns ?? EMPTY_ARRAY
+    (layoutHintsFrontColumns: ColumnName[] | undefined) =>
+      layoutHintsFrontColumns ?? []
```

Comment:
This was intentionally using `EMPTY_ARRAY` as a const, so that at a higher level it was memoized properly... though upon further reflection, I guess in this case it doesn't matter because it'll only return the `[]` when `layoutHintsFrontColumns` is `undefined`, and the memoize here will keep that object cached whenever `undefined` is passed; so disregard my first sentence and well done :)